.github/workflows/podman-e2e-windows.yaml

```diff
@@ -144,7 +145,7 @@ jobs:
         name: E2E-results-windows-${{ matrix.windows-version }}${{ matrix.windows-featurepack }}
         path: |
           podman-e2e-results-*.xml
-          podman-desktop-e2e-results-*.xml
-
+          results/podman-desktop-e2e-results-*.xml
```

Comment:
Looking at https://github.com/containers/podman-desktop-e2e/blob/main/.github/workflows/podman-desktop-e2e-windows.yaml#L198 are you sure that the junit file will be in the results directory? Not in $WORKSPACE or pwd when collecting the artifacts?